src/UIKit/UIGestureRecognizer.cs

```diff
@@ -159,163 +175,52 @@ public IEnumerable<Token> GetTargets ()
 #if !TVOS
 	public partial class UIRotationGestureRecognizer : UIGestureRecognizer {
 		public UIRotationGestureRecognizer (Action action) : base (action) {}
-		public UIRotationGestureRecognizer (Action<UIRotationGestureRecognizer> action) : base (Selector.GetHandle (UIGestureRecognizer.parametrized_selector), new Callback (action)) {}
+		public UIRotationGestureRecognizer (Action<UIRotationGestureRecognizer> action) : base (Selector.GetHandle (UIGestureRecognizer.parametrized_selector), new Callback<UIRotationGestureRecognizer>(action)) {}
```

Comment:
```suggestion
		public UIRotationGestureRecognizer (Action<UIRotationGestureRecognizer> action) : base (Selector.GetHandle (UIGestureRecognizer.parametrized_selector), new Callback<UIRotationGestureRecognizer> (action)) {}
```
Mono guidelines are used in the project: https://www.mono-project.com/community/contributing/coding-guidelines/ 

PS: I'm not a fan, I hate that extra space.